src/fdk_organization_bff/service/adapter.py

```diff
@@ -211,9 +211,12 @@ async def fetch_org_dataset_catalog_scores(
     """Fetch rating for organization's dataset catalog from fdk-metadata-quality-service."""
     url = f"{Config.metadata_uri()}/api/scores"
 
-    scores = await fetch_json_data_with_post(url, {"datasets": uris}, session)
-
-    if scores and isinstance(scores, Dict):
-        return scores
-    else:
+    if len(uris) > 500:
```

Comment:
Hva med evt ta de første 500

Reply:
vi viser bare en total, inntrykket blir da at det er total score for hele katalogen, men det gjelder bare 500

Reply:
hvis vi skal gjøre det så må vi ha en obsobs i frontend, og total score er uansett feil, for den gjelder bare et uttrekk. Jeg ser ikke hva verdien er i informasjonen da